packages/http-cache-storage/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@comunica/http-cache-storage",
```

Comment:
I guess we don't need a separate package for this one.
We can just move the interface to `@comunica/types`.